src/pages/Cases/Cases.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import Servicos from '../../components/Servicos/Servicos';
 
 const Cases: React.FC = () => {
   return (
     <div>
       <h1>Cases</h1>
-      <h1>Teste</h1>
+      <Servicos />
```

Comment:
Deixa para adicionar os componentes nas páginas para quem for puxar essas tarefas.